plugin/src/main/java/io/snyk/eclipse/plugin/views/snyktoolview/SnykToolView.java

```diff
@@ -261,6 +281,75 @@ public void toggleIgnoresButtons() {
 
 	}
 
+	/*
+	 * Sets up for doing a Net New Issues scan.
+	 */
+	public void enableDelta() {
+		clearDeltaNodeListeners();
+
+		Display.getDefault().asyncExec(() -> {
+			if (this.treeViewer != null && !this.treeViewer.getTree().isDisposed()) {
+
+				TreeItem[] rootItems = getTreeViewer().getTree().getItems();
+				for (TreeItem item : rootItems) {
+					ContentRootNode node = (ContentRootNode) item.getData();
+					String projectName = node.getName();
+					String projectPath = node.getPath().toString();
+					IProject project = ResourcesPlugin.getWorkspace().getRoot().getProject(projectName);
```

Comment:
Probably better to use the path here :)

Reply:
ResourcesPlugin.getWorkspace().getRoot().getProject(projectName) takes a name, I could not find a function that would return the project from the project name.

I could use IProject[] allProjects = workspace.getRoot().getProjects();
and loop through all the projects and compare with the path. 

for (IProject project : openProjects) {
Path path = ResourceUtils.getFullPath(project);
			...
			
			
Or do you have another way?